src/main/resources/assets/jei/lang/en_ud.lang

```diff
@@ -0,0 +1,116 @@
+jei.tooltip.config=ᵷᴉɟuoƆ IƎՐ
+jei.tooltip.show.recipes=sǝdᴉɔǝᴚ ʍoɥS
+jei.tooltip.show.all.recipes=sǝdᴉɔǝᴚ ꞁꞁⱯ ʍoɥS
+jei.tooltip.delete.item=ǝʇǝꞁǝᗡ oʇ ʞɔᴉꞁƆ
+jei.tooltip.liquid.amount.with.capacity=ᗺɯ p'% / p'%
+jei.tooltip.liquid.amount=ᗺɯ p'%
+jei.tooltip.transfer=sɯǝʇI ǝʌoW
+jei.tooltip.recipe.ore.dict=%s :ʎuɐ sʇdǝɔɔⱯ
+jei.tooltip.item.colors=%s :sɹoꞁoƆ
+jei.tooltip.shapeless.recipe=ǝdᴉɔǝᴚ ssǝꞁǝdɐɥS
+jei.tooltip.cheat.mode=ǝpoW ʇɐǝɥƆ
+jei.tooltip.recipe.by=%s :ʎᗺ ǝdᴉɔǝᴚ
+
+jei.tooltip.error.recipe.transfer.missing=sɯǝʇI ᵷuᴉssᴉW
+jei.tooltip.error.recipe.transfer.inventory.full=ꞁꞁnɟ ooʇ sᴉ ʎɹoʇuǝʌuI
+jei.tooltip.error.recipe.transfer.unknown=sᵷoꞁ ǝǝs 'ɹoɹɹƎ uʍouʞu∩
+jei.tooltip.error.recipe.transfer.no.server=pǝꞁꞁɐʇsuᴉ IƎՐ ǝʌɐɥ ʇsnɯ ɹǝʌɹǝs ǝɥ⟘
+jei.tooltip.error.recipe.transfer.too.large.player.inventory=˙pᴉɹᵷ ᵷuᴉʇɟɐɹɔ ɹǝʎɐꞁd ᘔxᘔ ǝɥʇ uᴉ ʇɟɐɹɔ oʇ ǝᵷɹɐꞁ ooʇ sᴉ ǝdᴉɔǝᴚ
+jei.tooltip.error.crash=ᵷoꞁ ǝǝs 'ɹoɹɹǝ dᴉʇꞁoo⟘
+
+jei.chat.error.command.too.long=˙puǝs oʇ ʇɟɐɹɔǝuᴉW ɹoɟ ᵷuoꞁ ooʇ sᴉ ʇᴉ 'puɐɯɯoƆ ʇɐɥƆ sᴉɥʇ ǝꞁpuɐɥ oʇ ɹǝʌɹǝs ǝɥʇ uo ǝq ʇsnɯ IƎՐ
+jei.chat.error.no.cheat.permission.1=˙ǝpoW ʇɐǝɥƆ s,IƎՐ ǝsn oʇ uoᴉssᴉɯɹǝd ǝʌɐɥ ʇou op no⅄
+jei.chat.error.no.cheat.permission.2=˙ǝpoW ǝʌᴉʇɐǝɹƆ uᴉ ǝɹɐ ɹo sɯǝʇᴉ ǝʌᴉᵷ/ uɐɔ oɥʍ sɹǝʎɐꞁd oʇ uǝʌᴉᵷ sᴉ uoᴉssᴉɯɹǝԀ
+
+key.jei.toggleOverlay=ʎɐꞁɹǝʌO ʇsᴉꞀ ɯǝʇI ǝꞁᵷᵷo⟘
+key.jei.focusSearch=ɹɐᗺ ɥɔɹɐǝS ʇɔǝꞁǝS
+key.jei.showRecipe=ǝdᴉɔǝᴚ ɯǝʇI ʍoɥS
+key.jei.showUses=sǝs∩ ɯǝʇI ʍoɥS
+key.jei.recipeBack=ǝdᴉɔǝᴚ pǝʍǝᴉᴧ ʎꞁsnoᴉʌǝɹԀ ʍoɥS
+key.jei.toggleCheatMode=pǝꞁqɐuƎ ᵷuᴉʇɐǝɥƆ ǝꞁᵷᵷo⟘
+key.jei.previousPage=ǝᵷɐԀ snoᴉʌǝɹԀ ʍoɥS
+key.jei.nextPage=ǝᵷɐԀ ʇxǝN ʍoɥS
+
+config.jei.default=ʇꞁnɐɟǝᗡ
+config.jei.valid=pᴉꞁɐᴧ
+
+config.jei.title=ᵷᴉɟuoƆ ƎWⱯNᗡO%M
```

Comment:
`%MODNAME` is a string that gets replaced in-code, it should not be made upside down.
I don't think there is an automatic way to detect replacement strings like this since it can be any format.

Reply:
Assuming this is made into a gradle plugin, you can add a special exceptions list when it runs in gradle

Reply:
@KnightMiner Not yet... It is still an applet. https://github.com/The-Fireplace/MC-en-UD-Generator